elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -139,4 +139,28 @@ default void flush() {
                                                           Optional<Sorting> sorting, Optional<Pagination> pagination) {
         return collection;
     }
+
+    default <T> Object getRelation(
+            Object entity,
+            RelationshipType relationshipType,
+            String relationName,
+            Class<T> relationClass,
+            EntityDictionary dictionary,
+            Set<Predicate> filters
+    ) {
+        return PersistentResource.getValue(entity, relationName, dictionary);
```

Comment:
The more I think about it, perhaps the dictionary is the appropriate place. I recommend moving `getValue` into the dictionary as a static method and using it from there rather than the `PersistentResource`. Any objections, speak now! :)